src/main/io/flashfs.c

```diff
@@ -645,8 +644,12 @@ bool flashfsVerifyEntireFlash(void)
     for (address = 0; address < testLimit; address += bufferSize) {
         tfp_sprintf(buffer, "%08x >> **0123456789ABCDEF**", address);
         flashfsWrite((uint8_t*)buffer, strlen(buffer), true);
+        if ((address % 0x10000) == 0) {
+            LED0_TOGGLE;
+        }
+        // Don't overwrite the buffer if the FLASH is busy writing
+        flashfsFlushSync();
```

Comment:
Normally flush should only be needed on write end - but that is with file / database systems :)